### ISSUE TITLE
AfterInsert event id is not set

### ISSUE THREAD

**Issue by vlorc** [go-xorm/xorm#749]
```
		sqlStr = sqlStr + " RETURNING " + session.Engine.Quote(table.AutoIncrement)
		res, err := session.query(sqlStr, args...)

		handleAfterInsertProcessorFunc(bean)
```

session_insert.go:448
session_insert.go:403

pg/oracle both is set id after the call AfterInsert

**Comment by vlorc** [comment]
#750

**Comment by lunny** [comment]
closed by #750
